src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -485,22 +543,46 @@ def setUp(self):
                 self.hostlist_clients, self.workdir,
                 self.hostfile_clients_slots)
 
+        # Access points to use by default when starting servers and agents
+        self.access_points = self.params.get(
+            "access_points", "/run/setup/*", self.hostlist_servers[:1])
+
         # Display host information
         self.log.info("--- HOST INFORMATION ---")
-        self.log.info("hostlist_servers:  %s", self.hostlist_servers)
-        self.log.info("hostlist_clients:  %s", self.hostlist_clients)
-        self.log.info("server_partition:  %s", self.server_partition)
-        self.log.info("client_partition:  %s", self.client_partition)
+        self.log.info("hostlist_servers:    %s", self.hostlist_servers)
+        self.log.info("hostlist_clients:    %s", self.hostlist_clients)
+        self.log.info("server_partition:    %s", self.server_partition)
+        self.log.info("client_partition:    %s", self.client_partition)
         self.log.info("server_reservation:  %s", self.server_reservation)
         self.log.info("client_reservation:  %s", self.client_reservation)
+        self.log.info("access_points:       %s", self.access_points)
 
-        # Kill commands left running on the hosts (from a previous test) before
-        # starting any tests.  Currently only handles 'orterun' processes, but
-        # can be expanded.
+        # List common test directory contents before running the test
+        self.log.debug("Common test directory (%s) contents:", self.test_dir)
         hosts = list(self.hostlist_servers)
         if self.hostlist_clients:
             hosts.extend(self.hostlist_clients)
-        self.stop_leftover_processes(["orterun"], hosts)
+        for line in get_file_listing(hosts, self.test_dir).stdout.splitlines():
+            self.log.debug("  %s", line)
+
+        if not self.start_servers_once or self.get_test_info()["id"] == 1:
+            # Kill commands left running on the hosts (from a previous test)
+            # before starting any tests.  Currently only handles 'orterun'
+            # processes, but can be expanded.
+            hosts = list(self.hostlist_servers)
+            if self.hostlist_clients:
+                hosts.extend(self.hostlist_clients)
+            self.stop_leftover_processes(["orterun"], hosts)
+
+            # Ensure write permissions for the daos command log files when
+            # using systemctl
+            if (self.agent_manager_class == "Systemctl" or
```

Comment:
I wonder if in general these kinds of references should be called `Systemd` rather than `Systemctl` which is the name of a tool for the manager.

Reply:
Possibly, though this is technically the name of the job_manager class and the other classes are named after the command they run.